__tests__/app.test.js

```diff
@@ -230,6 +228,16 @@ describe('App', () => {
           expect(article).toMatchObject(articleId1)
         })
     });
+    // comment count
+    test('200: article retrieved with id contains comment count property', () => {
+      return request(app)
+        .get("/api/articles/1")
+        .expect(200)
+        .then(({ body, body: { article } }) => {
+          expect(body).toHaveProperty("article")
+          expect(article).toHaveProperty("comment_count", 11)
```

Comment:
bravo for doing this as it's own test and not smuggling it in with the previous article_id one!